packages/web-components/src/menu-item/README.md

```diff
@@ -0,0 +1,126 @@
+# Menu Item
+
+Menu list item options displayed in a Menu component. They are invoked when users interact with a button, action, or other control.
+
+<hr />
+
+<br />
+
+**Remaining work items**
+
+1. Ensure divider component is styled appropriately in the context of a Menu
+2. Create support for menu item "grouping"
+3. Split button variation
```

Comment:
If these are still valid, create tracking issues, with Fluent UI WC (v3) tag.

fwiw, FAST already supports grouping by using a divider item between groups. Do we want to achieve parity with FUIR9 which uses `name` for the groups or are we OK with the difference?

Reply:
1 (divider). This has been confirmed the divider is being used in the storybook

[Issue #27042:  Menu Closing Behavior and Support for Menu Grouping, Header, and Split Button to MenuItem Component](https://github.com/microsoft/fluentui/issues/27042)